zipline/pipeline/loaders/_13d_filings.py

```diff
@@ -0,0 +1,54 @@
+"""
+Reference implementation for 13d filings loaders.
+"""
+
+from zipline.pipeline.common import (
+    DISCLOSURE_DATE,
+    PERCENT_SHARES,
+    NUM_SHARES
+)
+from zipline.pipeline.data import _13DFilings
+from zipline.pipeline.loaders.events import EventsLoader
+from zipline.utils.memoize import lazyval
+
+
+class _13DFilingsLoader(EventsLoader):
+    """
+    Reference loader for
+    :class:`zipline.pipeline.data._13DFilings`.
+
+    events_by_sid: dict[sid -> pd.DataFrame(knowledge date,
+    disclosure date, percent shares, number of shares)]
+
+    """
+    expected_cols = frozenset([DISCLOSURE_DATE,
+                               PERCENT_SHARES,
+                               NUM_SHARES])
+    event_date_col = DISCLOSURE_DATE
```

Comment:
What does having this column as the `event_date_col` do? Was this not something that was needed before that we recently added?